conformance/results/kong-kubernetes-ingress-controller.yaml

```diff
@@ -0,0 +1,55 @@
+typemeta:
+  kind: ""
+  apiversion: ""
+implementation:
+  organization: kong
+  project: kubernetes-ingress-controller
+  url: github.com/kong/kubernetes-ingress-controller
+  version: "2.10"
+  contact:
```

Comment:
The API states the following:
```
	// Contact is contact information for the maintainers so that Gateway API
	// maintainers can get ahold of them as needed. Ideally this should be
	// Github usernames (in the form of `@<username>`) or team names (in the
	// form of `@<team>/<name>`), but when that's not possible it can be email
	// addresses.
	Contact []string `json:"contact"`
```

But by reading [GH documentation](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams#team-visibility), it looks like people who are not members of the organization cannot view any teams. If it is the case, I don't think that giving the possibility to insert here teams makes much sense.

I am curious if @shaneutt has any thoughts on this one.